apps/site/lib/site_web/controllers/search_controller.ex

```diff
@@ -76,21 +76,18 @@ defmodule SiteWeb.SearchController do
     json(conn, %{error: "bad_response"})
   end
 
-  @spec log_error({:ok, HTTPoison.Response.t()} | {:error, HTTPoison.Error.t() | atom}) ::
-          :ok | {:error, any}
   def log_error({:ok, %HTTPoison.Response{} = response}) do
-    do_log_error(response)
+    do_log_error(response.body)
   end
 
   def log_error({:error, %HTTPoison.Error{} = response}) do
-    do_log_error(response)
+    do_log_error(response.reason)
   end
 
   def log_error(_) do
     :ok
   end
 
-  @spec do_log_error(HTTPoison.Response.t() | HTTPoison.Error.t()) :: :ok | {:error, any}
   defp do_log_error(error) do
     Logger.warn("Received bad response from Algolia: #{inspect(error)}")
```

Comment:
This inspect was the problem because it was printing out the headers of the HTTPoison.Response which is where the Algolia keys were. So, I just pulled the body out of the response so we have some information on what was sent. For an HTTPoison.Error we grab the reason.